resources/eventing/charts/nats/templates/statefulset.yaml

```
@@ -58,10 +58,16 @@ spec:
 {{- end }}
       # Common volumes for the containers.
       volumes:
+      ### the secret that holds account data ###
+      {{- if and .Values.auth.enabled .Values.auth.resolver }}
+      - name: resolver-volume
+        secret:
+          secretName: {{ .Values.auth.resolver.configMap.name }}
```

```suggestion
          secretName: {{ .Values.auth.resolver.secret.name }}
```

Related to: https://github.com/kyma-project/kyma/pull/15274/files#r959474476